### PR TITLE
Migrate user email

### LINKED ISSUE #27: Implement migration email template and strings
From https://github.com/mozilla/id.webmaker.org/issues/109

Need to implement an email template/strings for sending migration emails to migrating users on id.webmaker.org (from passwordless login.webmaker.org).

### PR DESCRIPTION
**Pull request by cadecairos**
Fixes #27

### CODE REVIEW

**Comment by cadecairos** [comment]
test with npm test and gulp dev (navigate to "http://localhost:8000/migrate_user.html")

**Comment by cadecairos** [comment]
@secretrobotron quick r?

**Comment by secretrobotron** [comment]
Needs a test for `gulp test`.

**Comment by secretrobotron** [comment]
"click here" in the text should be a link

**Comment by secretrobotron** [comment]
any way to make the button direct to a server dependent on an environment variable?

File: locale/en_US/migrate_user.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "subject_migrate_user": {
```
Comment (secretrobotron) [inline locale/en_US/migrate_user.json]: Where does this get used?

**Comment by secretrobotron** [comment]
@cadecairos few things to look at here. As well, I suggested some copy changes in https://github.com/mozilla/id.webmaker.org/issues/109#issuecomment-90892608 . Doesn't make enough sense yet, IMO.

File: locale/en_US/migrate_user.json (hunk 1 above)
Comment (cadecairos) [inline locale/en_US/migrate_user.json]: It's used to localize the subject of the email here: https://github.com/mozilla/webmaker-mailroom/blob/1080fa97753e5cfc055e37b674ad532f5cb89d7a/index.js#L62

Comment (secretrobotron) [inline locale/en_US/migrate_user.json]: :+1:

**Comment by secretrobotron** [comment]
@cadecairos just got the :+1: from @KevZawacki. Let's go with the copy in https://github.com/mozilla/id.webmaker.org/issues/109#issuecomment-90892608

**Comment by cadecairos** [comment]
> any way to make the button direct to a server dependent on an environment variable?

@secretrobotron The url is determined by the app triggering the call to createToken on the loginAPI, so that the email ends up pointing at the right server.

**Comment by secretrobotron** [comment]
Great! :+1: 

@KevZawacki @vazquez can you take a quick look at this? I think it's uglier than the original, but it does the job for now. R?

screenshot 2015-04-08 11 14 44

**Comment by secretrobotron** [comment]
@cadecairos can you just change the first bit of copy to `Webmaker is using a new login system to ensure<br>your experience is simple and safe.`? (note the `<br>`)

After that, r+. Gogogogo.

**Comment by KevZawacki** [comment]
Looks good!

**Pull request merged**